src/tiled/templatesdock.cpp

```diff
@@ -180,4 +349,6 @@ void TemplatesView::onPressed(const QModelIndex &index)
 
     if (ObjectTemplate *objectTemplate = model->toObjectTemplate(index))
         emit currentTemplateChanged(objectTemplate);
+    else
+        emit currentTemplateChanged(nullptr);
```

Comment:
Yes, I just said the condition is no longer needed, because it will do that also when you just write:
```c++
ObjectTemplate *objectTemplate = model->toObjectTemplate(index);
emit currentTemplateChanged(objectTemplate);
```